src/models/transactions.js

```diff
@@ -1,7 +1,28 @@
-const { Schema, model } = require("mongoose");
+const { Schema, model } = require('mongoose');
 
-const transaction = new Schema({}, { versionKey: false, timestamps: true });
+const transaction = new Schema(
+  {
+    type: {
+      type: String,
+      enum: ['-', '+'],
+    },
+    category: {
+      type: String,
+    },
+    comment: {
+      type: String,
+    },
+    sum: {
+      type: Number,
```

Comment:
u nich to nie jest number ale string.. nie jestem pewien Czy będzie wysyłany string czy number.